x/vm/module.go

```diff
@@ -127,6 +127,11 @@ func (AppModule) Name() string {
 func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterMsgServer(cfg.MsgServer(), am.keeper)
 	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)
+
+	m := keeper.NewMigrator(am.keeper)
+	if err := cfg.RegisterMigration(types.ModuleName, 9, m.Migrate9to10); err != nil {
```

Comment:
Same as above - I want to avoid having migrations for modules before v1

Reply:
Done!